.coveragerc

```diff
@@ -7,3 +7,6 @@ omit =
     *_test.py
     *core/tests/*
     *__init__.py
+    # TODO(lilithxxx): Remove python_utils from the list once the codebase is
```

Comment:
Please make an issue instead and use TODO(#ISSUENUM)

Reply:
We have to keep TODOs in this format: https://github.com/oppia/oppia/blob/564b7081737f2f3bd4026be980a392c322bc727a/scripts/pre_commit_linter.py#L155

I have mentioned the issue https://github.com/oppia/oppia/issues/7419 in the comment though

Reply:
Nope this has been fixed, take a look:
https://github.com/oppia/oppia/blob/7959c9d2ce81f1c0d40e26b361ba6a03e7bab3c7/core/templates/dev/head/domain/topic/SubtopicPageContentsObjectFactory.ts#L42

Reply:
EDIT: Used TODO(#ISSUENUM)